docs/index.md

```diff
@@ -0,0 +1,86 @@
+# Terraform AWS Provider - Contributor Guide
+
+The AWS Provider is the work of thousands of contributors, and is maintained by a small team within HashiCorp. There are many way to contribute, and we have extensive documentation on them. Below are the main ways most users engage with the provider repository and the high level steps required to ensure the contribution goes smoothly.
+
+## What would you like do do?
+
+### Add a New AWS Service to the Provider
+
+Allow Terraform (via the AWS Provider) to manage an entirely new AWS service by introducing the resources and datasources required to manage configuration of the service.
+
+- [Add Service Client](add-a-new-service.md#add-a-service-client)
+- [Perform Service Design](add-a-new-service.md#perform-service-design)
+- [Add a New Resource](add-a-new-service.md#add-a-new-resource)
```

Comment:
This link seems to just go to the "Add a service" page; it doesn't seem to have an "Add a new resource" header